### PR TITLE
Ksubrama/testkitchen

### PR DESCRIPTION
**Pull request by ksubrama**

### CODE REVIEW

File: chefignore [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+.colander
+.strainer
+
+# Vagrant #
+###########
+.vagrant
+Vagrantfile
+
+# Travis #
+##########
```
Comment (jdmundrawala) [inline chefignore]: might want .kitchen in this file

File: .kitchen.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  name: winrm
+
+provisioner:
+  name: chef_zero
+
+platforms:
+  - name: windows-8.1-chef11.10.4
+    driver_config:
+      box: windows-8.1
+      box_url: http://cdn.box-cutter.com/windows/virtualbox4.3.20/eval-win81x64-enterprise-nocm-1.0.2.box
```
Comment (jdmundrawala) [inline .kitchen.yml]: Get rid of the broken links

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+
+group :development do
+  gem "test-kitchen", :git => 'https://github.com/test-kitchen/test-kitchen.git', :branch => 'windows-guest-support'
+  gem 'kitchen-vagrant', git: 'https://github.com/afiune/kitchen-vagrant.git', :branch => 'Transport'
+  gem "berkshelf"
+  gem "vagrant-wrapper", ">= 2.0"
```
Comment (jdmundrawala) [inline Gemfile]: Since you've added chefspec stuff, you probably should probably add rspec and chefspec in here

**Comment by jdmundrawala** [comment]
Looks good, had a few minor comments

**Comment by ksubrama** [comment]
Done!  I believe chefspec should bring rspec in.

**Comment by smurawski** [comment]
Other than @jdmundrawala's comments, :+1:

**Comment by EasyAsABC123** [comment]
:+1: can't wait to have this so we can have some full iis regression tests

**Pull request merged**